### PR TITLE
added page and components for home

### PR DESCRIPTION
**Pull request by anupkgurung**

### CODE REVIEW

File: src/components/Cateogory/Cateogory.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+export const Cateogory = () => {
+    const [error, setError] = useState();
+    const [cateogory, setCateogory] = useState();
+
+    useEffect(()=>{
+        (async ()=>{
+            try{
+                await axios.get("api/categories")
+                .then(({ data : {categories} })=>{
```
Comment (Aditi0505) [inline src/components/Cateogory/Cateogory.jsx:13]: No need to use .then if you are using async-await. You can wrap it in try catch block to handle error, and directly use the response object you get from the await call.

File: src/components/Cateogory/Cateogory.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        <div id="aside" className="aside-container">
+            <aside className="padding-top-right">
+                <h5 className="text-color">Browse by Category</h5>
+                <ul className="cateogory">
+                    {cateogory && cateogory.map(({ _id, categoryName }) => (
+                        <li className="m-1 list-style" key={_id}>
+                            <Link className="link-style-none" to="/">{categoryName}</Link>
+                        </li>
+                    ))}
+                    {/* href="./page/productlist.html */}
```
Comment (Aditi0505) [inline src/components/Cateogory/Cateogory.jsx:32]: Please remove commented code if not needed.

File: src/components/Navbar/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                    <input type="text" className="search input-width" placeholder="Search" />
+                </div>
+                <div>
+                    <ul>
+                        <li className="list-inline">
+                            <span className="material-icons pl-1">account_circle</span>
+                        </li>
+                        <li className="list-inline">
+                            <Link className="link-style-none" to="/">
+                                {/* href="./wishlist.html */}
```
Comment (Aditi0505) [inline src/components/Navbar/Navbar.jsx:25]: Please remove commented code if not needed.

File: src/components/Navbar/Navbar.jsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                                <span className="material-icons pl-1">favorite_border</span>
+                            </Link>
+                        </li>
+                        <li className="list-inline">
+                            <Link className="link-style-none" to="/">
+                                {/* href="./cart.html" */}
```
Comment (Aditi0505) [inline src/components/Navbar/Navbar.jsx:31]: Please remove commented code if not needed.

File: src/components/Navbar/navbar.css [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    padding: 0rem 25rem;
+}
+.navbar-list > li {
+    padding: 1rem;
+}
+.nav {
+    padding: 0rem;
+}
+.nav-box-shadow {
+    box-shadow: -1px 5px 5px -4px black;
```
Comment (Aditi0505) [inline src/components/Navbar/navbar.css:40]: Try to use hex codes instead.

File: src/components/Product/VerticleProductCard.jsx [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+
+export const VerticleProductCard = () => {
+    const [productList, setProducts] = useState();
+
+    useEffect(() => {
+        (async () => {
+            try {
+                await axios.get("/api/products")
+                    .then(({ data }) => {
```
Comment (Aditi0505) [inline src/components/Product/VerticleProductCard.jsx:12]: Same as above

File: src/components/Product/VerticleProductCard.jsx [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+                        const { products } = data;
+                        setProducts(products);
+                    })
+
+            } catch (error) {
+                console.log(error);
```
Comment (Aditi0505) [inline src/components/Product/VerticleProductCard.jsx:18]: Display the error on UI instead.

File: src/pages/Home/home.css [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+    display: grid;
+    grid-template-columns: 25% 75%;
+    grid-gap: 2rem;
+    margin-top: 7rem;
+    padding: 2rem 1rem;
+}
+.aside-container {
+    border-right: 1px solid #00000038;
+    flex-basis: 35rem;
+    /* box-shadow: 2px 0px 3px 0px black; */
```
Comment (Aditi0505) [inline src/pages/Home/home.css:20]: Remove commented code

File: src/pages/Home/home.css [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+    padding: 0;
+}
+.card-img-container {
+    height: 12rem;
+}
+.card-desc {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    /* gap: 2rem; */
```
Comment (Aditi0505) [inline src/pages/Home/home.css:70]: Please remove commented code.

File: src/pages/Home/home.css [hunk 10]
```diff
@@ -0,0 +1,134 @@
...
+    border-radius: 0;
+    
+}
+.product-card-footer .btn-secondary-outline {
+    border-bottom-left-radius: 5px;
+    border-bottom-right-radius: 5px;
+    margin: 0.01rem;
+}
+.category li input,
+.price li input {
```
Comment (Aditi0505) [inline src/pages/Home/home.css:98]: Can use a common class for both here

File: src/components/Navbar/Navbar.jsx [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+            <div className="search-bar-nav">
+                <input type="text" className="search input-width" placeholder="Search" />
+            </div>
+            <div id="navigation" className="navigation-bar">
+                <nav className="nav nav-box-shadow">
+                    <ul className="ml-1 navbar-list">
+                        <li><Link className="link-style-none" tabIndex="1" to="/">Books</Link></li>
+                        <li><Link className="link-style-none" tabIndex="1" to="/">New Arrivals</Link></li>
+                        <li><Link className="link-style-none" tabIndex="1" to="/">Best Seller</Link></li>
+                        <li><Link className="link-style-none" tabIndex="1" to="/">Pre-Order</Link></li>
```
Comment (Aditi0505) [inline src/components/Navbar/Navbar.jsx:52]: There is a lot of repetition here. You can map over a list here.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions to look at. Rest seems good to me.

File: src/components/Navbar/Navbar.jsx (hunk 11 above)
Comment (TheAmlanSaikia) [inline src/components/Navbar/Navbar.jsx:52]: You can use .map to avoid repetition.

File: src/components/Navbar/navbar.css [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+    padding: 0.4rem 0.5rem 0.3rem 0.45rem;
+}
+.logo{
+    height: 4rem;
+    width: 7rem;
+}
+.navbar-list {
+    display: flex;
+    width: 100%;
+    justify-content: space-around;
```
Comment (TheAmlanSaikia) [inline src/components/Navbar/navbar.css:30]: You can use utility classes for flex and flex properties.

**Review by TheAmlanSaikia — APPROVED** [review]
Awesome work.  Left a single comment.

**Pull request merged**